tests/qam-minimal/update_minimal.pm

```diff
@@ -21,24 +21,18 @@ sub run {
 
     capture_state('between-after');
 
-    assert_script_run("zypper -n lr | grep test-minimal");
+    my $ret = zypper_call("lr | grep test-minimal");
+    die "zypper failed with code $ret" unless $ret == 0;
```

Comment:
I don't think you need to move *every* zypper call to the util function. This is rather a grep call